pkg/database/openshiftclusters.go

```diff
@@ -144,6 +144,9 @@ func (c *openShiftClusters) patch(key string, f func(*api.OpenShiftClusterDocume
 }
 
 func (c *openShiftClusters) Update(doc *api.OpenShiftClusterDocument) (*api.OpenShiftClusterDocument, error) {
+	if doc.LeaseOwner != c.uuid {
```

Comment:
No backend worker should call Update() - only Patch().  This needs to go into Patch somehow.

Reply:
Patch has it too. Do we see a case where ANY process should call `Update()` without a lease and should succeed?

Reply:
Yes.  The (only) place where Update is called outside of Patch is in the front end in response to an incoming PUT.  The front end doesn't use leases.

Reply:
So this is probably broken in the case of >1 RP behind LB.